MdePkg/Include/IndustryStandard/Tdx.h

```diff
@@ -113,8 +113,8 @@ typedef struct {
 typedef struct {
   UINT64    Gpaw;
   UINT64    Attributes;
-  UINT32    MaxVcpus;
   UINT32    NumVcpus;
+  UINT32    MaxVcpus;
   UINT64    Resv[3];

```

Comment:
Do  you need to add VcpuIndex?

Reply:
No need yet. It just needs to get correct NumVcpus because TDVF uses this info to accept memory in parallel. 
VcpuIndex can be added when it actually gets used in TDVF in the future.